tensorboard/webapp/app_routing/internal_utils_test.ts

```diff
@@ -108,6 +108,20 @@ describe('app_routing/utils', () => {
       expect(actual).toEqual(['1', '2']);
     });
 
+    it('returns single id from card route', () => {
+      const actual = utils.getExperimentIdsFromRouteParams(RouteKind.CARD, {
+        experimentIds: '1',
```

Comment:
nit: could you use a number that more like a test? e.g. "111" or "222", which also makes it more stands out on search.